docs/content/installation/installation-with-operator.md

```diff
@@ -22,7 +22,7 @@ This document describes how to install the NGINX Ingress Controller in your Kube
     * To pull from the F5 Container registry, configure a docker registry secret using your JWT token from the MyF5 portal by following the instructions from [here](/nginx-ingress-controller/installation/using-the-jwt-token-docker-secret).
     * It is also possible to build your own image and push it to your private Docker registry by following the instructions from [here](/nginx-ingress-controller/installation/building-ingress-controller-image).
 2. Install the NGINX Ingress Operator following the [instructions](https://github.com/nginxinc/nginx-ingress-helm-operator/blob/v1.3.0/docs/installation.md).
-3. Create the default server secret and SecurityContextConstraint as outlined in the [instructions](https://github.com/nginxinc/nginx-ingress-helm-operator/blob/v1.3.0/docs/installation.md).
+3. Create the default server secret and SecurityContextConstraint as outlined in the ["Getting Started" instructions](https://github.com/nginxinc/nginx-ingress-helm-operator#getting-started).
```

Comment:
Why are we removing the tag here?

Reply:
The document linked was completely wrong: are all Helm Operator doc links meant to be tagged?

Reply:
yeah we always use tags in the links

Reply:
@lucacome I'll push an update for the tag, but what's the strategy for if the tag changes? I don't have a mental image of how tags are being used by Helm Operator.

e.g If there's X amount of tagged links from NIC docs, and a new version of Helm Operator comes out, suddenly all of those tagged links are invalid. Suddenly, there could be drift in the documentation.

Reply:
All the links to the operator repo are on this page and we should update them with every new version of the operator. The problem with using `main` is that we could point to something that is not yet released.

We have a table here with all the versions https://github.com/nginxinc/kubernetes-ingress/blob/main/docs/content/technical-specifications.md#supported-kubernetes-versions

Reply:
Tagged link added. I noticed there's 1.3.1 now, but I stuck with 1.3.0 since that's what the other links go to.